src/Windows/Markdown/Editor/Impl/Preview/Margin/PreviewMargin.cs

```diff
@@ -92,13 +115,25 @@ public sealed class PreviewMargin : Border, IWpfTextViewMargin, IMarkdownPreview
             _textChanged = true;
         }
 
+        private void UpdateBrowserScrollPosition() {
+            if(!_settings.ScrollPreviewWithEditor) {
+                return;
+            }
+
+            IdleTimeAction.Create(() => {
+                var lineNumber = _textView.TextSnapshot.GetLineNumberFromPosition(_textView.Caret.Position.BufferPosition);
```

Comment:
Generally no. OnIdle only runs when browser pane is active while UpdateBrowserScrollPosition runs when caret position changes which means editor pane has focus.